### ISSUE TITLE
Improve verbose output when using --delete-excluded

### ISSUE THREAD

**Issue by diamondsw** [rclone/rclone#878]
> What is your rclone version (eg output from `rclone -V`)
1.34
> Which OS you are using and how many bits (eg Windows 7, 64 bit)
macOS 10.12.1, 64-bit
> Which cloud storage system are you using? (eg Google Drive)
Amazon Drive
> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
/usr/local/bin/rclone copy --verbose --stats 0 --dry-run --exclude-from /Users/jochs/Desktop/rclone_filter.txt --delete-excluded "/Volumes/Shares/Archive" "amazon:/Backup/Archive"
> A log from the command with the `-v` flag (eg output from `rclone -v copy /tmp remote:tmp`)
.DS_Store: Excluded from sync (and deletion)                                                                                                                    
.VolumeIcon.icns: Excluded from sync (and deletion)                                                                                                             
Media/.DS_Store: Excluded from sync (and deletion)                                                                                                              
Computing/.DS_Store: Excluded from sync (and deletion)                                                                                                          
Documents/.DS_Store: Excluded from sync (and deletion)                                                                                                          

In short, rclone still prints "(and deletion)" even with the '--delete-excluded' option. This makes it sound as if excluded files won't be deleted, although I expect they actually would be. Holding off on using this option until it's clarified. ;)

**Comment by vesper8** [comment]
+1 I agree this should be changed, I use --delete-excluded and got confused by this message in the log

**Comment by ncw** [comment]
@ssaqua fancy fixing up the log messages?

**Comment by ssaqua** [comment]
Sure, will start a PR soon.

**Comment by tgallant** [comment]
looks like this can be closed
